main.bib

```diff
@@ -4114,6 +4114,146 @@ @inproceedings{nyatsanga2023comprehensive
   evaluation = {qualitative, human},
   other = {n/a},
 }
+@misc{yang_social_2024,
```

Comment:
Could you update https://github.com/eliclaggett/awesome-social-agents/blob/113ae90d5eb471d48cf47c47e0146f58bd2bf73f/main.bib#L31C10-L31C24 instead of adding a new entry?

Reply:
Oops, so sorry about that. I thought I checked for duplicates but that one slipped through.